core/src/main/resources/filodb-defaults.conf

```diff
@@ -171,18 +171,21 @@ filodb {
     translate-prom-to-filodb-histogram = true
     # Timeout for query engine subtree/ExecPlans for requests to sub nodes
     # Higher default until we have a way to really timeout query execution at leaves
-    ask-timeout = 120 seconds
+    ask-timeout = 30 seconds
 
     stale-sample-after = 5 minutes
 
     # Maximum number of samples to return in a query
     sample-limit = 1000000
 
     # Binary Join Cardinality limit
-    join-cardinality-limit = 25000
+    join-cardinality-limit = 100000
 
     # Group by Cardinality limit
-    group-by-cardinality-limit = 1000
+    group-by-cardinality-limit = 100000
+
+    # Maximum result size returnable from any ExecPlan::execute call (including nested calls)
+    result-byte-limit = 1000000000
```

Comment:
These were previously unused-- their now-used values are pulled from the `PlannerParams` constructor.

Reply:
What would happen to an instant aggregated query with high churn?I understand this is a very large number but something to callout for? Rather than throwing an exception, I will give a flexibility to either throw an exception or log a warning. This would make it easy to see which unexpected queries which worked previously now start (or would start) failing. This would give a change to fine tune these limits without impacting the end users across versions.

Reply:
Excellent point-- I've added a config flag that indicates whether-or-not the limit should be enforced. Regardless of enforcement, there is now always a warning logged if the limit is breached.